packages/test/src/copy-fixture.js

```diff
@@ -1,12 +1,14 @@
 import fs from "fs-extra";
 import path from "path";
+import pkgDir from "pkg-dir";
 import replaceStream from "replacestream";
 import * as constants from "./constants";
 
 export default copyFixture;
```

Comment:
This file is named `copyFixture.js` because it provides the `copyFixture` function. Not sure why the dash-cased rename?

Reply:
I'll revert this